spec/packaging/ibm_cloud_pak/qualification-yaml.md

```diff
@@ -65,20 +75,88 @@ catalog.visible|boolean|If true (the default) if not present, this IBM Cloud Pak
 
 # Examples
```

Comment:
It is my understanding that if only OpenShift chart won't include psps - is that still true?  If yes, does it make sense to further clarity example is for chart that runs on both ICP & OpenShift?